lib/warder/coffee_lint_runner/message_formatter.rb

```diff
@@ -0,0 +1,19 @@
+require 'warder/coffee_lint_runner/message_line_formatter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.